.github/workflows/android-nightly.yml

```
@@ -0,0 +1,34 @@
+  # Allow manual runs from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  all-brands:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)